### PR TITLE
zh: update TOC

### PR DESCRIPTION
**Pull request by toutdesuite**
### What is changed, added or deleted? (Required)

Update TOC

### Which TiDB Data Migration (DM) version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v2.0 changes for now)
- [ ] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s):

### CODE REVIEW

File: zh/TOC.md [hunk 1]
```diff
@@ -5,62 +5,66 @@
...
-  - [同步延迟监控](feature-overview.md#同步延迟监控)
-  - [Online-ddl-scheme](feature-online-ddl-scheme.md)
-  + Shard Support
-    - [简介](feature-shard-merge.md)
-    - [使用限制](feature-shard-merge.md#使用限制)
-    - [手动处理 Sharding DDL Lock](feature-manually-handling-sharding-ddl-locks.md)
-+ Benchmark
-  - [DM 1.0-GA 性能测试](benchmark-v1.0-ga.md)
-+ 使用场景
++ TiDB Data Migration 简介
```
Comment (yikeke) [inline zh/TOC.md:8]:
```suggestion
+ 关于 TiDB Data Migration
```
To unify the word usage with https://github.com/pingcap/docs-cn/blob/docs-special-week/TOC.md

File: zh/TOC.md [hunk 2]
```diff
@@ -5,62 +5,66 @@
...
+  + 基本信息
+    - 开源
+    - 性能数据
+  - [主要特性](feature-overview.md) 参考 [Online-ddl-scheme](feature-online-ddl-scheme.md)，和 Shard Support 相关文档（[简介](feature-shard-merge.md)，[使用限制](feature-shard-merge.md#使用限制)，[手动处理 Sharding DDL Lock](feature-manually-handling-sharding-ddl-locks.md)）
```
Comment (yikeke) [inline zh/TOC.md:12]:
```suggestion
  - [主要特性](key-features.md) 参考 [Online-ddl-scheme](feature-online-ddl-scheme.md)，和 Shard Support 相关文档（[简介](feature-shard-merge.md)，[使用限制](feature-shard-merge.md#使用限制)，[手动处理 Sharding DDL Lock](feature-manually-handling-sharding-ddl-locks.md)）
```

File: zh/TOC.md [hunk 3]
```diff
@@ -5,62 +5,66 @@
...
-  + Shard Support
-    - [简介](feature-shard-merge.md)
-    - [使用限制](feature-shard-merge.md#使用限制)
-    - [手动处理 Sharding DDL Lock](feature-manually-handling-sharding-ddl-locks.md)
-+ Benchmark
-  - [DM 1.0-GA 性能测试](benchmark-v1.0-ga.md)
-+ 使用场景
++ TiDB Data Migration 简介
+  + 基本信息
+    - 开源
```
Comment (yikeke) [inline zh/TOC.md:10]:
```suggestion
    - [开源信息说明](licensing.md)
```
ref: https://github.com/pingcap/docs-cn/blob/docs-special-week/TOC.md

File: zh/TOC.md [hunk 4]
```diff
@@ -5,62 +5,66 @@
...
+    - 性能数据
+  - [主要特性](feature-overview.md) 参考 [Online-ddl-scheme](feature-online-ddl-scheme.md)，和 Shard Support 相关文档（[简介](feature-shard-merge.md)，[使用限制](feature-shard-merge.md#使用限制)，[手动处理 Sharding DDL Lock](feature-manually-handling-sharding-ddl-locks.md)）
+  - What's New
```
Comment (yikeke) [inline zh/TOC.md:13]:
```suggestion
  - What's New
```
DM 这边确定现在要维护这个页面吗？这个应该是需要定期更新的。我觉得如果需要暂时不大，可以把这个页面暂时去掉 @WangXiangUSTC @csuzhangxc

Comment (csuzhangxc) [reply zh/TOC.md:13]: 我觉得可以去掉

Comment (yikeke) [reply zh/TOC.md:13]: Ok. 帮忙 commit suggestion 下吧～ @toutdesuite

Comment (toutdesuite) [reply zh/TOC.md]: updated~

File: zh/TOC.md [hunk 5]
```diff
@@ -5,62 +5,65 @@
...
++ [常见问题](faq.md)
 + 版本发布历史
   + v1.0
     - [1.0.2](releases/1.0.2.md)
     - [1.0.3](releases/1.0.3.md)
     - [1.0.4](releases/1.0.4.md)
     - [1.0.5](releases/1.0.5.md)
-- [TiDB DM 术语表](glossary.md)
++ [术语表](glossary.md)
++ 文档指南
```
Comment (lilin90) [inline zh/TOC.md:69]:
```suggestion
```

File: zh/TOC.md [hunk 6]
```diff
@@ -5,62 +5,64 @@
...
-    - [简介](feature-shard-merge.md)
-    - [使用限制](feature-shard-merge.md#使用限制)
-    - [手动处理 Sharding DDL Lock](feature-manually-handling-sharding-ddl-locks.md)
-+ Benchmark
-  - [DM 1.0-GA 性能测试](benchmark-v1.0-ga.md)
-+ 使用场景
++ 关于 TiDB Data Migration
+  + 基本信息
+    - [开源信息说明](licensing.md)
+    - 性能数据
```
Comment (csuzhangxc) [inline zh/TOC.md:11]: 能咨询一下这个 `性能数据` 和后面的 `性能测试` 的主要区别是什么吗

Comment (WangXiangUSTC) [reply zh/TOC.md:11]: 一个是告诉用户如何测试性能，一个是写下我们的测试数据给用户参考 🐶

Comment (csuzhangxc) [reply zh/TOC.md:11]: OK

Comment (yikeke) [inline zh/TOC.md:11]:
```suggestion
    - [性能数据](performance.md)
```

File: zh/TOC.md [hunk 7]
```diff
@@ -5,62 +5,64 @@
...
+  - [主要特性](key-features.md)  可参考 [特性概述](feature-overview.md)，[Online-ddl-scheme](feature-online-ddl-scheme.md)，和 Shard Support 相关文档（[简介](feature-shard-merge.md)，[使用限制](feature-shard-merge.md#使用限制)，[手动处理 Sharding DDL Lock](feature-manually-handling-sharding-ddl-locks.md)）
+  - 应用场景 参考 [分表合并数据迁移最佳实践](shard-merge-best-practices.md)，[DM-worker 在上游 MySQL 主从间切换](usage-scenario-master-slave-switch.md)
+  - 荣誉列表
+  - 售后支持
```
Comment (yikeke) [inline zh/TOC.md:15]:
```suggestion
```
Removed for now. Consider adding them back later.

File: zh/TOC.md [hunk 8]
```diff
@@ -5,62 +5,64 @@
...
+    - 查询 [参考](manage-replication-tasks.md#查询数据同步任务状态)
+    - 暂停 [参考](manage-replication-tasks.md#暂停数据同步任务)
+    - 重启
+    - 删除
+  - 告警处理
+  - 日常巡检
++ 故障处理 参考 [故障诊断](troubleshoot-dm.md)，[错误含义](error-system.md)
+  - [故障及处理方法](error-handling.md)
+  - 性能问题及处理方法
++ 开发者指南
```
Comment (yikeke) [inline zh/TOC.md:46]:
```suggestion
```
Removed for now. Consider adding them back later.

**Pull request merged**